components/script/dom/document.rs

```diff
@@ -247,6 +248,9 @@ pub struct Document {
     referrer: Option<String>,
     /// https://html.spec.whatwg.org/multipage/#target-element
     target_element: MutNullableHeap<JS<Element>>,
+    /// https://w3c.github.io/uievents/#event-type-dblclick
+    #[ignore_heap_size_of = ""]
```

Comment:
Yeah, it seems that `std::time` types are not supported on heapsize. We might want to, can you open an issue in https://github.com/servo/heapsize, and use `#[ignore_heap_size_of = "<url-to-the-issue>"]`, or similar? Thanks!